gobblin-core/src/main/java/gobblin/source/extractor/extract/kafka/SimpleKafkaSource.java

```diff
@@ -0,0 +1,37 @@
+package gobblin.source.extractor.extract.kafka;
+
+import java.io.IOException;
+
+import com.google.common.base.Preconditions;
+
+import gobblin.configuration.WorkUnitState;
+import gobblin.source.extractor.Extractor;
+import gobblin.util.ClassAliasResolver;
+import gobblin.util.reflection.GobblinConstructorUtils;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+
+/**
+ * A {@link KafkaSource} to use with arbitrary {@link KafkaExtractor}. Specify the extractor to use with key
+ * {@link #EXTRACTOR_TYPE}.
+ */
+public class SimpleKafkaSource<S, D> extends KafkaSource<S, D> {
```

Comment:
We have another `KafkaSimpleSource` existing in the same package. Can you rename this class?